### ISSUE TITLE
_Nonnull not recognized

### ISSUE THREAD

**Issue by JanX2** [drkameleon/iswift#136]
When trying to convert the following:

		NSArray *URLs = @[];
		[URLs enumerateObjectsWithOptions:NSEnumerationConcurrent
										usingBlock:^(NSURL * _Nonnull thisURL, NSUInteger idx, BOOL * _Nonnull stop) {
			NSString *scheme = [thisURL scheme];
		}];

I get this error: “Unexpected :  #identifier ('thisURL')”

I have yet to find a workaround.

**Comment by drkameleon** [comment]
Well.... the quick workaround is to remove the `_Nonnull` parts (both in front of `thisURL` and `stop`). *Is this even valid? Honestly, it's the first time i'm seeing `_Nonnull` - i'll have to look it up in the reference)

**Example:**

```objective-c
@implementation aClass

- (void)aFunc {
    NSArray *URLs = @[];
    [URLs enumerateObjectsWithOptions:NSEnumerationConcurrent
                                    usingBlock:^(NSURL * thisURL, NSUInteger idx, BOOL *  stop) {
        NSString *scheme = [thisURL scheme];
    }];
}
@end
```

**Comment by JanX2** [comment]
`_Nonnull`appears to be new in Xcode 7.1.

**Comment by drkameleon** [comment]
Aha... ooook... perfectly valid (and perfectly... not supported yet by iSwift): https://developer.apple.com/swift/blog/?id=25

I'm putting in my to-do list. Thanks for letting me know! ;-)

**Comment by drkameleon** [comment]
Issue fixed as of the upcoming 2.2.

----

Pending: Properly convert `_Nonnull` and `_Nullable` to optional/non-optional vars respectively.

**Comment by drkameleon** [comment]
https://github.com/drkameleon/iswift/issues/137
